.github/workflows/test-webservice.yml

```diff
@@ -28,8 +29,8 @@ jobs:
         xcode-version: latest
     - uses: actions/cache@v2
       with:
-        path: .build
-        key: ${{ runner.os }}-spm-cachev2-${{ hashFiles('Package.resolved') }}
+        path: TestWebService/.build
```

Comment:
Is the cache action not impacted by the `working-directory` definition?

Reply:
Yes, it seems like it. The previous version didn't work.
And regarding the timeout: It is defined seconds.

Reply:
Weird behavior 😅 

Would it make sense to increase that even further? Should boot in 5 seconds, but we also have seen GitHub Actions being really slow 🙃 But lets just see how it goes.

Reply:
Yes, I could also imaging extending the timeout even more if this is needed. On the other hand this gives us some hint about changes that dramatically increase the startup time of the web service so I would suggest to leave it at the current timeout for now and increase it once we have the need for it or we notice that builds start failing due to a timeout.